src/components/FileUpload.js

```diff
@@ -246,6 +291,11 @@ class FileUpload extends React.Component {
               </div>
             ) : (
               <div style={styles.centered}>
+              {this.state.failedValidationTypes.length > 0 ? (
+                  this.state.failedValidationTypes.map(validationType => {
+                    return (<div style={styles.invalidMessage}>{this._getDefaultValidationMessage(validationType)}</div>);
+                  })
+              ) : null }
```

Comment:
It's silly but we don't put a space after `null`. I'd only change it though if you're already making another change.